comment/messages.py

```diff
@@ -86,3 +90,13 @@ class FlagState:
 class FollowError:
     EMAIL_REQUIRED = _('Email is required to subscribe {model_object}')
     SYSTEM_NOT_ENABLED = _('Subscribe system must be enabled')
+
+
+class BlockState:
+    UNBLOCKED = _('Unblocked')
+    BLOCKED = _('Blocked')
+
+
+class BlockUserMSG:
```

Comment:
We already have a class in this module `ErrorMessage`, maybe we can be more consistent by using `BlockUserMessage`

Reply:
It should be `BlockUserError` here.